htdocs/core/modules/propale/doc/pdf_azur.modules.php

```diff
@@ -753,6 +840,26 @@ function write_file($object,$outputlangs,$srctemplatepath='',$hidedetails=0,$hid
 					}
 				}
 
+                //var_dump($pagenb);exit;
+                // output all grids
+                foreach ($arraynotegrids as $key => $grid) {
```

Comment:
On action adding line you will have a warning like " Invalid argument supplied for foreach()  " because $arraynotegrids will not be set.